src/core_modules/capture-core/components/WidgetIndicator/WidgetIndicatorContent/sortIndicatorsFn.js

```diff
@@ -1,13 +1,18 @@
-// eslint-disable-next-line complexity
+/* eslint-disable complexity */
+import { localeCompareStrings } from '../../../utils/localeCompareStrings';
+
 export const sortIndicatorsFn = (a, b) => {
     if (typeof b === 'string') {
-        return (((a.key && a.key.localeCompare(b)) || (a.message && a.message.localeCompare(b))));
+        return ((a.key && localeCompareStrings(a.key, b)) || (a.message && localeCompareStrings(a.message, b)));
     }
     if (b.message) {
-        return (((a.key && a.key.localeCompare(b.message)) || (a.message && a.message.localeCompare(b.message))));
+        return (
+            (a.key && localeCompareStrings(a.key, b.message)) ||
+            (a.message && localeCompareStrings(a.message, b.message))
+        );
     }
     if (b.key) {
-        return (((a.key && a.key.localeCompare(b.key)) || (a.message && a.message.localeCompare(b.key))));
+        return ((a.key && localeCompareStrings(a.key, b.key)) || (a.message && localeCompareStrings(a.message, b.key)));
     }
     return 1;
 };
```

Comment:
@eirikhaugstulen: Can you have a second look at this function when you find the time (in another PR). It seems like we are missing some logic for typeof a === 'string' and the result could depend on whether an item is on the a or b side.